### PR TITLE
Fix #85

### LINKED ISSUE #85: Supporting Content Empty: The chat endpoint does not have content for supporting content. This may be related to sources not existing.
MicrosoftTeams-image

### PR DESCRIPTION
**Pull request by LittleLittleCloud**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: app/frontend/Components/Answer.razor [hunk 1]
```diff
@@ -59,7 +59,7 @@
                          ToolTip="Show the supporting content." Disabled="@(Retort is { DataPoints: null } or { DataPoints.Length: 0 })">
                 <ChildContent>
                     <MudPaper Class="pa-2" Elevation="3">
-                        <SupportingContent DataPoints="Retort.DataPoints" />
+                        <SupportingContent DataPoints="Retort.DataPoints  " />
```
Comment (luisquintanilla) [inline app/frontend/Components/Answer.razor:62]: Is this extra space supposed to be here?

**Review by luisquintanilla — APPROVED** [review]
Thanks @LittleLittleCloud. Other than my comment it looks good.

File: app/frontend/Components/Answer.razor (hunk 1 above)
Comment (luisquintanilla) [reply app/frontend/Components/Answer.razor:62]: @LittleLittleCloud

Comment (LittleLittleCloud) [reply app/frontend/Components/Answer.razor:62]: Oh Nope, let me revert it

**Review by luisquintanilla — APPROVED** [review]

**Pull request merged**